packages/core/actions/account-client/sendSignedUserOperation.ts

```diff
@@ -0,0 +1,36 @@
+import { type SendUserOperationParameters, deepHexlify } from "permissionless"
+import type { SmartAccount } from "permissionless/accounts"
+import type { BundlerRpcSchema } from "permissionless/types/bundler"
+import type {
+    EntryPoint,
+    GetEntryPointVersion
+} from "permissionless/types/entrypoint"
+import type { UserOperationWithBigIntAsHex } from "permissionless/types/userOperation"
+import type { Chain, Client, Hash, Transport } from "viem"
+
+export async function sendSignedUserOperation<
```

Comment:
We can just use `eth_sendUserOperation` directly by extending it on kernelClient itself, right? Seems like a redundant action